models/get_prediction.py

```diff
@@ -81,13 +81,29 @@ def get_prediction_binary(model_dict_path, model_path, X):
 
     return y
 
+def get_prediction_parquet(model_dict_path, model_path, X_path):
```

Comment:
Is this not used anywhere? Shouldn't it be used to replace a large portion of the code in lines 141-175 of this file?